src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesLease.cs

```diff
@@ -66,7 +66,7 @@ public KubernetesLease(ExtendedActorSystem system, AtomicBoolean leaseTaken, Lea
             var client = new KubernetesApiImpl(system, kubernetesSettings);
             _timeout = _settings.TimeoutSettings.OperationTimeout;
             _leaseName = MakeDns1039Compatible(settings.LeaseName);
-            _leaseActor = system.ActorOf(
+            _leaseActor = system.SystemActorOf(
```

Comment:
Actual fix, make sure the `LeaseActor` lives under `/system`, not `/user`

Reply:
LGTM